README.md

```diff
@@ -1 +1,130 @@
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio] <a name="about-project"></a>
+
+
+**[Portfolio]** is a project for showcasing my skills and understanding in html and ccs flexbox.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Responsive]**
+- **[Toggle Bar Display]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+``` 
+    1. Git installed on your machine.
+    2. Any code editor [vscode preferred].
+    3. Any browser [chrome preferred].
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd my-folder
+  git clone git@github.com:CaptainBawa/Portfolio.git
+
+```
+
+### Install
+
+Install this project with:
+
+```
+  nmp install/npm i
+
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+  Live Server
+  
+```
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Collins Bawa**
+
+- GitHub: [@githubhandle](https://github.com/CaptainBawa)
+- Twitter: [@twitterhandle](https://twitter.com/BawaCollins)
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [1] **[Make Table Version]**
+- [2] **[Make Desktop Version]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+Dear Viewers!
+
+I'm excited to be working on this project and I would be grateful for your support! By supporting this project, you are not only helping me, but you are also contributing to something meaningful that can make a difference. Your support will give me the motivation and resources necessary to keep moving forward with this project and to ensure its success. So, whether it's through your kind words, your financial support, or simply by spreading the word, your support means everything to me. Thank you in advance for your support, and I can't wait to share with you the amazing things we will achieve together.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://www.mit.edu/~amini/LICENSE.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [OPTIONAL] Please🙏 remove the 'NOTE' section since it was a recommendation given to you regarding which license to use. This will make your README file look cleaner👍, with only valid information